extension-manifest-v3/src/content_scripts/adblocker-scriptlets.js

```diff
@@ -0,0 +1,5 @@
+// Should only be needed on Safari:
+// the tabId of the initial chrome.webNavigation.onCommitted
+// is not reliable. When opening bookmarks, it can happen that
+// the event is associated with a tabId of 0.
+chrome.runtime.sendMessage({ action: 'onCommitted' });
```

Comment:
As this is used only for adblocker scriptlets, what do you think about renaming that event to something more specific like `injectScriplets` or `resolveScriptlets`?

Reply:
good idea!